README.markdown

```diff
@@ -327,7 +327,7 @@ class TestDatabase : Database {
 }
 ```
 
-* Long lines should be wrapped at around 70 characters. A hard limit is intentionally not specified.
+* Long lines should be wrapped at around 250 characters. A hard limit of 400 is enforced.
```

Comment:
120 characters.

Reply:
I was always taught 120 chars however with everyone having widescreens these days, this could be set higher than 120. What do you think @johnrogers? > 120 makes it more readable

Reply:
Just checked what I have mine set to and its 160 which doesn't seem outrageous to me.